### ISSUE TITLE
thumbnails missing

### ISSUE THREAD

**Issue by gjost** [densho/ddr-public#45]
Sample: http://ddr.densho.org/ddr/densho/201/255/

**Comment by gjost** [comment]
From the page source:
```
<a href="http://ddr.densho.org/media/ddr-densho-201/ddr-densho-201-255-mezzanine-75b27c67f6-a.jpg" class="lightbox img-thumbnail" data-plugin-options="{"type":"image"}">
<img src="http://ddr.densho.org/media/cache/24/63/2463a5cb214001848355003028a857b8.jpg" class="img-responsive">
</a>
```
* access: http://ddr.densho.org/media/ddr-densho-201/ddr-densho-201-255-mezzanine-75b27c67f6-a.jpg
* thumb: http://ddr.densho.org/media/cache/24/63/2463a5cb214001848355003028a857b8.jpg

The access file is present and visible.  A thumbnail URL has been generated.  Another thumb is present in the `/media/cache/24/63/` and it is visible.

**Comment by gjost** [comment]
`/etc/ddr/local.conf` looks good:
```
[public]
...
# Base URL for collection media, to be inserted into templates.
media_url_local=http://192.168.0.30/media/
media_url=http://ddr.densho.org/media/
download_url=http://ddr.densho.org/download/media/
```

**Comment by gjost** [comment]
Not sure if related, but:
```
>>> from ui.models import Collection, Entity, File
>>> repo = 'ddr'; org = 'densho'; cid = '201'; eid = '255'; role = 'master'; sha1 = '0eed712b43'
>>> c = Collection.get(repo, org, cid)
>>> e = Entity.get(repo, org, cid, eid)
>>> f = File.get(repo, org, cid, eid, role, sha1)
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/usr/local/src/ddr-public/ddrpublic/ui/models.py", line 671, in get
    return build_object(File(), id, document['_source'])
  File "/usr/local/src/ddr-public/ddrpublic/ui/models.py", line 346, in build_object
    contents_display = field_display_handler[style](fieldname, contents)
  File "/usr/local/src/ddr-public/ddrpublic/ui/models.py", line 264, in display_filesize
    return t.render(c)
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/template/base.py", line 209, in render
    return self._render(context)
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/template/base.py", line 917, in render_node
    return node.render(context)
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/template/base.py", line 957, in render
    output = self.filter_expression.resolve(context)
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/template/base.py", line 674, in resolve
    new_obj = func(obj, *arg_vals)
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/template/defaultfilters.py", line 905, in filesizeformat
    value = ugettext("%s MB") % filesize_number_format(bytes / MB)
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/template/defaultfilters.py", line 892, in <lambda>
    filesize_number_format = lambda value: formats.number_format(round(value, 1), 1)
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/utils/formats.py", line 162, in number_format
    get_format('DECIMAL_SEPARATOR', lang, use_l10n=use_l10n),
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/utils/formats.py", line 110, in get_format
    for module in get_format_modules(lang):
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/utils/formats.py", line 82, in get_format_modules
    modules = _format_modules_cache.setdefault(lang, list(iter_format_modules(lang, settings.FORMAT_MODULE_PATH)))
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/utils/formats.py", line 51, in iter_format_modules
    if not check_for_language(lang):
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/utils/translation/__init__.py", line 181, in check_for_language
    return _trans.check_for_language(lang_code)
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/utils/lru_cache.py", line 125, in wrapper
    result = user_function(*args, **kwds)
  File "/usr/local/src/env/ddrpublic/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py", line 409, in check_for_language
    if not language_code_re.search(lang_code):
TypeError: expected string or buffer
>>> 
```

**Comment by gjost** [comment]
Ran `python manage.py thumbnail cleanup`.  No difference.

**Comment by gjost** [comment]
Ran `python manage.py thumbnail clear`. No effect.

**Comment by gjost** [comment]
Note: `ddrstage` is also running the `namesdb` branch (minus a few commits) and has the same problem.

**Comment by gjost** [comment]
Switched `ddrstage` to `develop` branch (thumbnails were working before all this namesdb stuff) and it does the same thing.

**Comment by gjost** [comment]
Looks like the `manage.py shell` behavior above is related to a Django bug (https://code.djangoproject.com/ticket/24569) that only manifests itself in the REPL.  So it's not related(?) to thumbnails but makes it hard to test.

**Comment by gjost** [comment]
Looks like it was `settings.MEDIA_ROOT`.  `Makefile` now has dynamic media in `/var/www/ddrpublic/media` but `settings.py` had not been updated to go along with it.

`MEDIA_ROOT` is now set to come from `/etc/ddr/ddr.cfg` or `local.cfg`.

Fixed in c29613b.
